country-a-service/testing-shared/src/main/java/dk/nsp/epps/testing/shared/Fmk.java

```diff
@@ -7,6 +7,14 @@
  */
 public class Fmk {
     private static final String fmkEndpointUri = "https://test2-cnsp.ekstern-test.nspop.dk:8443/decoupling";
+    /**
+     * Helle Bonde is a test persona which we do *not* own, so we should only perform read operations on her.
+     */
+    public static final String cprHelle = "1111111118";
```

Comment:
For good measure, it should probably also be in the variablename, like "cprHelleReadOnly"

Reply:
Good idea